pyro/infer/svi.py

```diff
@@ -40,6 +40,10 @@ def __init__(self,
                  num_samples=10,
```

Comment:
Can we add a similar warning and default for `num_samples`? Maybe we should change both defaults to `None` or "deprecated".

Reply:
Yes, good idea. Will fix.